ultralytics/nn/modules/head.py

```diff
@@ -28,6 +28,7 @@ class Detect(nn.Module):
     shape = None
     anchors = torch.empty(0)  # init
     strides = torch.empty(0)  # init
+    format = "_"
```

Comment:
@ambitious-octopus Hey buddy can we eliminate this line? since we don't have it in Detect along the way and nothing seems wrong to me. :) Also set it equals to `_` looks a bit strange.

Reply:
Hey @Laughing-q, we will have an attribute error if we remove this. Probably we should set this to `None`. 
Try to remove this line and then:
```
from ultralytics import YOLO
model = YOLO("yolov8n.pt")
model.predict()
```

Reply:
@ambitious-octopus ok I see! Thanks! I'll take a look tomorrow. :)